cmd/build.go

```diff
@@ -72,7 +72,7 @@ func buildCommandAction(cmd *cobra.Command, args []string) error {
 	}
 	logger.Debugf("Use build directory: %s", buildDir)
 
-	targets, err := docs.UpdateReadmes(packageRoot)
+	targets, err := docs.UpdateReadmes(packageRoot, "")
```

Comment:
Could `buildDir` be used here as that folder has been created above? 

```suggestion
	targets, err := docs.UpdateReadmes(packageRoot, buildDir)
```

Reply:
Umm, seeing that actually maybe we don't need all the magic below to create the build dir, as is being created explicitly here. Maybe I can remove some more code.